app/javascript/Pages/components/Button.scss

```diff
@@ -0,0 +1,71 @@
+$primary-color: #6600cc;
+$dark-color: #252636;
+$white-color: #ffffff;
+$gray-color: #aaaaaa;
+
+.btn {
+  padding: 0.5rem 1.25rem;
+  border-radius: 5px;
+  outline: none;
+  border: none;
+  cursor: pointer;
+  white-space: nowrap;
+}
+
+.btn--primary {
+  background-color: var($white-color);
+  color: $primary-color;
+  border: 1px solid var($white-color);
+}
+
+.btn--outline {
+  background-color: transparent;
+  color: $white-color;
+  padding: 0.5rem 1.25rem;
+  border: 1px solid var($white-color);
+  transition: all 0.3s ease-out;
+}
+
+.btn--medium {
+  padding: 0.5rem 1.25rem;
+  font-size: 1.1rem;
+  margin-right: 2rem;
+}
+
+.btn--large {
+  padding: 0.75rem 1.6rem;
+  font-size: 1.25rem;
+}
+
+.btn--mobile {
+  text-align: center;
+  border-radius: 4px;
+  width: 80%;
+  text-decoration: none;
+  font-size: 1rem;
+  background-color: transparent;
+  color: $white-color;
+  padding: 0.8rem 1.25rem;
+  border: 1px solid $white-color;
+  transition: all 0.3s ease-out;
+  margin-right: 1rem;
+}
+
+.btn--wide {
+  padding: 0.75rem 4rem;
+  font-size: 1.25rem;
+}
+
+.btn--large:hover,
+.btn--medium:hover,
+.btn--mobile:hover {
+  transition: all 0.3s ease-out;
+  background: $white-color;
+  color: $primary-color;
+}
+
+.btn--wide:hover {
```

Comment:
**suggestion (non-blocking)**: this can be placed inside the `btn--wide` class with `&:hover`